static/css/theme-tugboat.css

```diff
@@ -189,6 +189,11 @@ a:hover {
 #sidebar a:hover {
     color: var(--MENU-SECTIONS-LINK-HOVER-color);
 }
+
+#sidebar ul.topics {
+    margin: 1rem 1rem 0 1rem;
```

Comment:
I snuck this in, it's been bothering me a while. 😬

Reply:
I actually can't see what this is doing, so... go for it!